Extension/package.nls.json

```diff
@@ -977,7 +977,7 @@
     "c_cpp.walkthrough.set.up.title": "Set up your C++ Environment",
     "c_cpp.walkthrough.activating.description": "Activating the C++ extension to determine whether your C++ Environment has been set up.\nActivating Extension...",
     "c_cpp.walkthrough.no.compilers.description": "We could not find a C++ compiler on your machine, which is required to use the C++ extension. Follow the instructions on the right to install one, then click “Find my new Compiler” below.\n[Find my new Compiler](command:C_Cpp.RescanCompilers?%7B%22sender%22%3A%22walkthrough%22%7D)",
-    "c_cpp.walkthrough.compilers.found.description": "The C++ extension works with a C++ compiler. Select one from those already on your machine by clicking the button below.\n[Select my Default Compiler](command:C_Cpp.SelectDefaultCompiler?%7B%22sender%22%3A%22walkthrough%22%7D)",
+    "c_cpp.walkthrough.compilers.found.description": "The C++ extension works with a C++ compiler. Select one from those already on your machine by clicking the button below.\n[Select my Default Compiler](command:C_Cpp.SelectIntelliSenseConfiguration?%22walkthrough%22)",
```

Comment:
I think we kept it that way because we couldn't check the status of the experiment in the walkthrough.  I believe we intended to migrate the walkthrough to use the new Select IntelliSense Configuration, but maybe we do that in a separate PR?

Reply:
>Otherwise, users could choose "Select a compiler" and they could see configuration providers and compile commands as well.

@sean-mcmanus We added a check that if select IntelliSense configuration is invoked from the walkthrough, then only show the compilers and not configuration providers and compile commands. There was redundant code between select compiler and select IntelliSense configuration, so it was consolidated into select IntelliSense configuration. @Colengms and I looked at it last night and decided on this merged.